projects/components/src/datagrid/filters/datagrid-filter.ts

```diff
@@ -64,7 +64,10 @@ export abstract class DatagridFilter<V, C extends FilterConfig<V>, F extends For
 {
     abstract formGroup: F;
 
-    protected constructor(filterContainer: ClrDatagridFilter, private subscriptionTracker: SubscriptionTracker) {
+    protected constructor(
+        protected filterContainer: ClrDatagridFilter,
+        private subscriptionTracker: SubscriptionTracker
```

Comment:
Since we're touching this file, we may as well remove the providers from here since it doesn't have any effect, the concrete classes need to add the providers.

Reply:
@kepelrs discovered this while analyzing the SubscriptionTracker

Reply:
IIRC it was @irahov who discovered it first :)